test/functionalTest/harnessFunctions.sh

```diff
@@ -103,9 +103,12 @@ function dbDrop()
 {
   db=$1
 
-  if [ "$db" != "" ]
+  if [ "$CB_DB_DROP" != "No" ]
```

Comment:
You set it, if you want to avoid to drop the database (my comment in the previous PR):
% CB_DB_DROP=No testHarness.sh XXX.test